portlist/poller.go

```diff
@@ -17,7 +18,9 @@ import (
 	"tailscale.com/envknob"
 )
 
-var pollInterval = 5 * time.Second // default; changed by some OS-specific init funcs
+// PollInterval is the recommended OS-specific interval
+// to wait between *Poller.Poll method calls.
+var PollInterval = 5 * time.Second // default; changed by some OS-specific init funcs
```

Comment:
I'd rather not have this API be a mutable variable.

add a func for this (keeping the variable unexported), and document it as returning a "recommended poll interval for the OS"

Reply:
@bradfitz that's definitely nicer, done